src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3318,10 +3318,7 @@ Give the compiler some way to differentiate the methods. For example, you can gi
     <value>'id#' syntax is no longer supported. Use '$id' instead.</value>
   </data>
   <data name="WRN_DefineIdentifierRequired" xml:space="preserve">
-    <value>Invalid value for '/define'; '{0}' is not a valid identifier</value>
-  </data>
-  <data name="WRN_DefineIdentifierRequired_Title" xml:space="preserve">
-    <value>Invalid value for '/define'; not a valid identifier</value>
+    <value>Invalid value for a preprocessing symbol; '{0}' is not a valid identifier</value>
```

Comment:
It looks like the message became somewhat confusing, I believe it is the name of the symbol is incorrect, not its value.